### PR TITLE
Tile update for Varadero

### PR DESCRIPTION
**Pull request by Aurallianz**
## About the PR
Added Tiles Walls and a few objects for Varadero.

## Why / Balance
Need it for Varadero mapping.

## Media
- [x] I have added Media to this PR. 
image

video
**Changelog**

### CODE REVIEW

File: Resources/Prototypes/_RMC14/Entities/Structures/Base/base_structure.yml [hunk 1]
```diff
@@ -122,3 +122,40 @@
...
+        layer:
+        - HighImpassable
+        - MidImpassable
+        - LowImpassable
+        - InteractImpassable
+        density: 1000
+  - type: RMCDropshipBlocked
+
+
+
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Structures/Base/base_structure.yml:161]: Small nitpick, two too many new lines at the end of the file
```suggestion
  - type: RMCDropshipBlocked

```

File: Resources/Textures/_RMC14/Structures/Doors/Airlocks/strata_door.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,162 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken/modified from cmss13 at https://github.com/cmss13-devs/cmss13/blob/f7b9367cb5c083a8c052a40dece9417031e28ffe/icons/obj/structures/doors/strata/strata_doors.dmi",
```
Comment (DrSmugleaf) [inline Resources/Textures/_RMC14/Structures/Doors/Airlocks/strata_door.rsi/meta.json:4]: This link is invalid, it leads to a 404

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by Aurallianz** [comment]
I will not be adding any more to this PR.

**Review by DrSmugleaf — APPROVED** [review]
Thanks for the PR!

**Pull request merged**